bot/exts/fun/user_events.py

```diff
@@ -0,0 +1,603 @@
+        return "Not scheduled"
+
+    @staticmethod
+    def live() -> str:
+        """To indicate user event is live."""
+        return "Live"
+
+    @staticmethod
+    def scheduled(start_datetime: datetime, end_datetime: datetime) -> str:
+        """
+        To indicate user event is scheduled.
+
+        The following code returns(an example):
+
+        Scheduled on 7th November 2020,
+        from 17:30 UTC To 18:00 UTC
+        (24-hour format).
+        """
+        readable_date = (
+            f"{start_datetime.day}{DATE_PREFIX.get(start_datetime.day, 'th')} "
+            f"{list(calendar.month_name)[start_datetime.month]} {start_datetime.year}"
+        )
+        readable_time = (
+            f"from {start_datetime.time().strftime('%H:%M')} UTC "
+            f"To {end_datetime.time().strftime('%H:%M')} UTC\n(24-hour format)."
+        )
+        status = f"Scheduled on {readable_date},\n{readable_time}"
+        return status
+
+    @property
+    def developers_role(self) -> Role:
+        """Return guild developers role."""
+        return self.guild.get_role(Roles.verified)
+
+    @property
+    def user_event_ongoing_role(self) -> Role:
+        """Return guild user-event-ongoing role."""
+        return self.guild.get_role(Roles.user_event_ongoing)
+
+    @property
+    def guild(self) -> Guild:
+        """Return guild instance."""
+        return self.bot.get_guild(Server.id)
+
+    @property
+    def user_event_coord_channel(self) -> TextChannel:
+        """Return #user-events-coordinators channel."""
+        return self.bot.get_channel(Channels.user_event_coordinators)
+
+    @property
+    def user_event_announcement_channel(self) -> TextChannel:
+        """Return #user-events-announcement channel."""
+        return self.bot.get_channel(Channels.user_event_announcements)
+
+    @property
+    def user_events_list_channel(self) -> TextChannel:
+        """Return #user-events-list channel."""
+        return self.bot.get_channel(Channels.user_event_list)
+
+    @property
+    def user_event_voice_channel(self) -> VoiceChannel:
+        """Return #user-events-voice channel."""
+        return self.bot.get_channel(Channels.user_event_voice)
+
+    @staticmethod
+    def user_event_embed(event_name: str, event_description: str, organizer: Member, status: str) -> Embed:
+        """Embed representing a user event."""
+        embed = Embed(
+            title=f"{event_name}",
+            description=(
+                f"Organizer: {organizer.mention}\n\n"
+                f"{event_description}\n\n"
+                f"**Status:** {status}"
+            )
+        )
+        return embed
+
+    async def fetch_subscribers(self, message_id: int) -> list:
+        """Fetch reacted users to event message as subscribers."""
+        # Fetch the event message
+        message = await self.user_events_list_channel.fetch_message(message_id)
+
+        reaction = message.reactions[0]
```

Comment:
I think we should make sure is this right emoji. I know this isn't something really important, but sometimes something with ratelimits that bot can't add new reaction and somebody else add some other reaction etc.